scala-list.cfg

```diff
@@ -0,0 +1,6 @@
+# scala versions to build images for
+2.11.8
```

Comment:
No particular reason, just didn't see a reason not to cover the bases. Want me to delete all but 2.12.2?